GTMAppAuthSwift/Sources/GTMOAuth2KeychainCompatibility.swift

```diff
@@ -0,0 +1,335 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+// Ensure that we import the correct dependency for both SPM and CocoaPods since
+// the latter doesn't define separate Clang modules for subspecs
+#if SWIFT_PACKAGE
+import AppAuthCore
+import GTMSessionFetcherCore
+#else
+import AppAuth
+import GTMSessionFetcher
+#endif
+
+// Standard OAuth keys
+let oauth2AccessTokenKey = "access_token"
+let oauth2RefreshTokenKey = "refresh_token"
+let oauth2ScopeKey = "scope"
+let oauth2ErrorKey = "error"
+let oauth2TokenTypeKey = "token_type"
+let oauth2ExpiresInKey = "expires_in"
+let oauth2CodeKey = "code"
+let oauth2AssertionKey = "assertion"
+let oauth2RefreshScopeKey = "refreshScope"
```

Comment:
static

Reply:
These are not declared on a type and cannot be static.